e2e_playwright/st_fragment_basics.py

```diff
@@ -23,9 +23,9 @@
 # NOTE: We intentionally don't verify that values returned by these widgets work as
 # expected as doing so in this type of batch test would drastically increase the
 # boilerplate code required to write this. Instead, we rely on other tests to fully test
-# return values. We also don't test the camera_input, data_editor, and file_uploader
-# widgets as well as custom components here due to the disproportionate amount of work
-# required to do so.
+# return values. We also don't test the audio_input, camera_input, data_editor, and
```

Comment:
just for documentation purposes, @kajarenc and I tested it manually yesterday. It's probably a good idea to verify this and also state this explicitly in the PR description 🙂

Reply:
Wait, so what's the ask here? To confirm that `st.audio_input` doesn't work with fragments?

Reply:
it now works in fragments!